src/visual/Visualizer.js

```diff
@@ -86,6 +86,20 @@ export default class Visualizer {
     this._timer = null;
     this._step = null;
     this.clear();
+
+    this.paperDefaultValidateConnectionMethod = this.paper.options.validateConnection;
```

Comment:
Thank you so much for the review, it is very valuable!

I have followed this recommendation and it has been done in b29a2e2e10 commit